resources/web/wwi/Makefile

```
@@ -103,13 +114,13 @@ webots.lib.js:
 $(THREEJS_SOURCE):
 	@echo "# downloading $(THREEJS_URL)"
 	@mkdir -p $(DEPENDENCIES_PATH)
-	@wget -qN -O $@ $(THREEJS_URL)
+	@$(WGET) $@ $(THREEJS_URL)
 
 $(LOCAL_THREEJS_EXAMPLE_SOURCES):
 	$(eval FILENAME=$(@:$(DEPENDENCIES_PATH)/%=%))
 	@echo "# downloading three.js $@"
 	@mkdir -p $(dir $@)
-	wget -qN -O $@ $(THREEJS_EXAMPLE_URL)$(FILENAME)
+	$(WGET) $@ $(THREEJS_EXAMPLE_URL)$(FILENAME)
```

This line should be silenced:

```suggestion
	@$(WGET) $@ $(THREEJS_EXAMPLE_URL)$(FILENAME)
```